### ISSUE TITLE
Strange behavior after creating new iso (Manjaro JWM)

### ISSUE THREAD

**Issue by holmeslinux** [manjaro/manjaro-tools#193]
hi guys good night! i am having a problem creating the iso of the Manjaro JWM. yesterday updated the manjaro-tools (manjaro-tools-base and manjaro-tools-iso 0.11.1-2) and then i create an iso and everything ok. after boot noticed three strange things: a) change style of the root terminal; b) change the style of pcmanfm root and c) high consumption (cpu and ram) of compton and nm-applet (in https://forum.manjaro.org/index.php?topic=31669.msg259961#msg259961). before updating the manjaro-tools-base and manjaro-tools-iso that did not happen. @oberon2007 oriented:

> You have to get the config also to root's home, like in /root/.config
Not sure right now how to accomplish that. Maybe you'll have to add a custom root-overlay folder with its own skel?

i did this: /root/skel/.config and i created a new and this did not work. i decided to do downgrade of the manjaro-tools-base and manjaro-tools-iso, but gives an error:

```
[holmes@manjaro-xfce ~]$ buildiso -p jwm -b stable
[sudo] password for holmes: 
ls: cannot access 'jwm/*.set': No such file or directory
/usr/lib/manjaro-tools/util.sh: eval: line 39: syntax error near unexpected token `)'
/usr/lib/manjaro-tools/util.sh: eval: line 39: `		) is_buildset=true ;;'
sed: can't read jwm//etc/manjaro-tools/iso.d.set: No such file or directory
cat: /etc/manjaro-tools/iso.d/jwm.set: No such file or directory
cat: /etc/manjaro-tools/iso.d/jwm.set: No such file or directory
[holmes@manjaro-xfce ~]$
```
manjaro-tools-base 0.10.2-1 and manjaro-tools-iso 0.9.10-2.

now i do not know what to do! someone could give me a orientation?

**obs**.: the profile of Manjaro JWM is in https://github.com/holmeslinux/Manjaro-JWM.

vlw fwi, Holmes :)

**Comment by udeved** [comment]
Hi Holmes,

please update the manjaro-iso-profiles fork, integrate your jwm in community, your skel in desktop-settings and send pull requests.

**Comment by holmeslinux** [comment]
hello udeved,
necessarily jwm profile must be in manjaro git? 
my profile is very big!
is possible to create the iso my git?

vlw fwi, Holmes :)

**Comment by udeved** [comment]
Its possible, but then you are on your own.
The big stuff needs to be packaged, we can do that.
So in short, either you adopt to new way of doing things,or you do on your own without support.

**Comment by holmeslinux** [comment]
hello udeved,
ok, will make the necessary changes to adapt.
only need help to create the packages, because i have no experience in this and want to learn!
the @oberon2007 will help.

vlw fwi, Holmes :)

**Comment by udeved** [comment]
Just sent a pull request for the profiles first.
I will tell you if something has to be removed before merge.

Packaging is no problem. ;)

**Comment by holmeslinux** [comment]
then i will do pull the way it is...
you tell me what to remove ;)
thank

vlw fwi, Holmes :)

**Comment by holmeslinux** [comment]
udeved,
i do not know if i did right! in folder /home/holmes/manjaro-tools-iso-profiles/
git status
git add *
git commit -a -m "..."
git push -f origin master

vlw fwi, Holmes :)

**Comment by udeved** [comment]
This one went wrong.

No, please fork manjaro-tools-iso-profiles.
Add your profile ad send a pull request via github.
You push to your fork first.

**Comment by udeved** [comment]
And don't force push please. don't do git push -f

**Comment by holmeslinux** [comment]
kkkkkk (vixi)
already did that! sorry and now?
i'm here: http://www.auplod.com/u/uapdlo75d0b.png

vlw fwi, Holmes :)

**Comment by udeved** [comment]
No, do a fork please. 
We go through a intermediate fork repo, before it is gonna be merged in manjaro/manjaro-iso-profiles.


Click on the fork button, then you get a fork in your account.
Pull the fork, work in there, and then send pull request, it will show you diff what you added.

**Comment by udeved** [comment]
Reason is, I don't want the big stuff in main repo, it will stay there in history and grows the repo again.

**Comment by udeved** [comment]
Holmes, don't push nested git repo.

Do a fork please.

**Comment by holmeslinux** [comment]
udeved:
this? 
http://www.auplod.com/u/audpol75d10.png

vlw fwi, Holmes :)

**Comment by udeved** [comment]
Yes, push there and send pull request, then I tell you what to remove before merge.

**Comment by holmeslinux** [comment]
jejejejeje
i will do it now!
thank

vlw fwi, Holmes :)

**Comment by holmeslinux** [comment]
udeved,
i'm in doubt, see image
http://www.auplod.com/u/uldopa75d14.png

**Comment by udeved** [comment]
Don't a a repo there, you need to add you jwm files.

**Comment by udeved** [comment]
Everything in jwm-overlay/usr needs to be packaged.
Don't push it, only push jwm-overlay/etc files

**Comment by holmeslinux** [comment]
see if you are correct!

**Comment by udeved** [comment]
Holmes, do you know someone who could translate you from english to portugese?
I think we have a language problem.

**Comment by Ste74** [comment]
@holmeslinux , check other profile in desktop-setting repo .. more your settings is possible to split in this repo ... and you need to set gschema override if jwm use this ( i don't know jwm ) . 

> Everything in jwm-overlay/usr needs to be packaged.
Don't push it, only push jwm-overlay/etc files

**Comment by udeved** [comment]
ste, can you try italian? Perhaps Holmes understands with Italian more than google translation does produce eng->por?

**Comment by Ste74** [comment]
Non sò , se volete e può servire vediamo come è tradotto questo .. Le impostazioni dei vari profili sono state splittate tutto quello presente in usr/share deve essere costruito con pacchetti quindi si devono costruire pacchetti per icone e temi (esempio vedo che hai molti temi ed icone) puoi creare un pkgbuild per costruire chessò manjaro-jwm-icon-set .. in più se jwm usa gschemas bisogna estrapolare le chiavi con dconf e inserirle qua nel file 
https://github.com/manjaro/desktop-settings/blob/master/community/budgie/schemas/99_manjaro-settings.gschema.override
il nuovo manjaro tools usa queste impostazioni .. si deve limitare al massimo le personalizzazioni in modo da poter avere maggiore controllo sull'instabilità che potrebbe generare un'alta configurazione .. io non sò se jwm è simile a bswm , potresti prendere spunto da esso ..

**Comment by udeved** [comment]
Ste, can you add,that holmeslinux/manjaro-tools-iso-profiles need to have the files from his jwm repo?
But only jwm-overlay/etc/* except the skel. No more, and then send pull request.

**Comment by Ste74** [comment]
Ok .. Holmes , per inserire il profilo jwm in manjaro-tools-iso-profiles devi togliere la cartella skel da jwm-overlay/etc/* la skel ora con le dovute correzioni è in desktop-settings.

**Comment by philmmanjaro** [comment]
@holmeslinux: I removed you from the **maintainers** group. You lately pushed stuff into our repositories with the **force** switch. We changed a lot on how we maintain our profiles. Most of the stuff is now in packages as **settings-packages**. Therefore we are better of with a fork for now until your profile is again within our standard. As of now we have to many profiles not matching our needs.
